docs/client-metrics.md

```diff
@@ -115,6 +115,7 @@ The event stream is a log of events and the time they occurred while the user is
 * error.&lt;unexpected_origin&gt;.auth.1027 - a postMessage message was received from an unexpected origin.
 * error.&lt;image_url&gt;.profile.997 - a profile image could not load.
 * error.&lt;screen_name&gt;.&lt;service&gt;.998 - System unavailable.
+* error.no context.auth.1050 - an attempt is being made to write an account with no uid to localStorage.
```

Comment:
It should actually be `unknown context`, [with a space](https://github.com/mozilla/fxa-content-server/blob/6530352888ef77ca860d73c0485d96d8727939e6/app/scripts/lib/metrics.js#L475).  I could change the context to `user` so that one exists.